website/docs/plugin/framework/rpcs.mdx

```diff
@@ -0,0 +1,206 @@
+
+### _terraform apply_
+
+| Protocol RPCs                                                                                                                                                                 | Framework Functionality                                                                                                                                            |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [GetProviderSchema](#getproviderschema-rpc)                                                                                                                                   | `provider.Provider`, `resource.Resource`, and `datasource.DataSource` interface `Schema` method                                                                    |
+| [ValidateProviderConfig](#validateproviderconfig-rpc) / [ValidateResourceConfig](#validateresourceconfig-rpc) / [ValidateDataResourceConfig](#validatedataresourceconfig-rpc) | `provider.Provider`, `resource.Resource`, and `datasource.DataSource` interface `Schema` method validators, `ConfigValidators` method, and `ValidateConfig` method |
+| [ConfigureProvider](#configureprovider-rpc)                                                                                                                                   | `provider.Provider` interface `Configure` method                                                                                                                   |
+| [ReadResource](#readresource-rpc) / [ReadDataSource](#readdatasource-rpc)                                                                                                     | `resource.Resource` and `datasource.DataSource` interface `Read` method                                                                                            |
+| [PlanResourceChange](#planresourcechange-rpc)                                                                                                                                 | `resource.Resource` interface `Schema` method plan modifiers and `ModifyPlan` method                                                                               |
+| [ApplyResourceChange](#applyresourcechange-rpc)                                                                                                                               | `resource.Resource` interface `Create`, `Update`, or `Delete` method                                                                                               |
+
+## GetProviderSchema RPC
+
+### Overview
+
+![diagram: GetProviderSchema RPC Overview](/img/get-provider-schema-overview.png)
```

Comment:
Can we have HTML comments (or some other documentation added) for how to update these diagrams?